### PR TITLE
Add 2 parameters to glow to allow gl repo to use glow to generate its bindings.

### PR DESCRIPTION
**Pull request by shurcooL**
- `-out` flag specifies the output directory where the generated Go package will be placed; defaults to "gl".
- `-prefix` flag specifies a common import path prefix, it is used to set the desired import path for procaddr packages.
- Remove Makefile since the gl bindings will be generated via go generate directives inside the new gl repo.
- Move the task of `gofmt`ing the generated Go package into the generator via `exec.Command("go", "fmt", ...)`. This is because `gofmt`ing is a part of the generation process. It can be improved to avoid shelling out, but that's an optional implementation detail.

Resolves task 2 of issue #52.

Needed for go-gl/gl#9 to resolve go-gl/gl#8.

### CODE REVIEW

File: package.go [hunk 1]
```diff
@@ -64,6 +56,11 @@ func (pkg *Package) GeneratePackage() error {
 		}
 	}
 
+	// gofmt the generated .go files.
```
Comment (slimsag) [inline package.go]: This is super nice! But PR description should mention it.

**Comment by slimsag** [comment]
Aside from my comment, LGTM.

File: package.go (hunk 1 above)
Comment (shurcooL) [inline package.go]: Added bullet point to PR description.

File: package.go [hunk 2]
```diff
@@ -11,6 +12,7 @@ import (
 
 // A Package holds the typedef, function, and enum definitions for a Go package.
 type Package struct {
+	Prefix  string
 	Name    string
```
Comment (errcw) [inline package.go]: Perhaps some of these struct entries may now be removed if they are unused?

Comment (shurcooL) [inline package.go]: Sure. Either in a follow up PR, or feel free to push to this branch.

It looks like there are a couple that may be unused now.

Comment (shurcooL) [inline package.go]: I've checked, and there was not a single unused struct fields. They are all still used (mostly by `UniqueName()`).

I did fix a typo in a comment, but that's all.

Comment (shurcooL) [inline package.go]: @errcw Is there anything else this needs to be merged?

**Pull request merged**